index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="">
+</head>
+<body>
+	<fieldset>
+		<legend>Build a tree.</legend>
+		<input type="text" id="height" placeholder="number of lines."></input>
+		<input type="text" id="character" placeholder="character symbol"></input>
+		<button type="submit" id="button">Grow your tree.</button>
+	</fieldset>
+	<script src="quiz.js" type="text/javascript" charset="utf-8" async defer></script>
+</body>
+</html>
```

Comment:
As Github helpfully points out with the 🚫  symbol, per convention from þe olde POSIX days, all files should end with a blank line (newline character).  You can read more about it [here](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) if you're so inclined.